app/main.py

```diff
@@ -2,0 +1,53 @@
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        self.skills = ["JavaScript", "HTML", "CSS"]
+
+    def create_awesome_web_page(self) -> str:
+        print(f"{self.name} is creating a webpage...")
+        return "<h1>Hello world</h1>"
```

Comment:
You are overriding the `create_awesome_web_page` method in the `FrontendDeveloper` class, but you are not adding any additional logic inside. This is considered a bad practice. If you don't need to add any additional logic, you should not override the method.